wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx

```diff
@@ -78,51 +76,20 @@ const TokenList = (props: Props) => {
     const tokenSet: Set<string> = new Set();
     const tokens: Array<TokenConfig> = [];
 
-    const addToken = (tokenConfig: TokenConfig) => {
```

Comment:
This function will have different conditions for most cases below and that's why I don't think we need a separate function anymore, not much re-usability.